lib/mixlib/shellout.rb

```diff
@@ -170,7 +170,7 @@ class ShellOut
     #   cmd = Mixlib::ShellOut.new("apachectl", "start", :user => 'www', :env => nil, :cwd => '/tmp')
     #   cmd.run_command # etc.
     def initialize(*command_args)
-      @stdout, @stderr, @process_status = "", "", ""
+      @stdout, @stderr, @process_status = String.new(""), String.new(""), String.new("")
```

Comment:
Might want to comment that this is force mutable strings in preparation for ruby 4.0, because someone might attempt to revert this?

Reply:
yeah we probably should. adding that now. thanks!